lib/activity_reporter.rb

```diff
@@ -26,7 +26,7 @@ def output(log_files = default_log_files)
   def extract_druid(file, today, counter)
     File.readlines(file).each do |line|
       line =~ /#{today}/ || next
-      line =~ %r{/bundle/ruby|/usr/local/rvm|resque-signals/} && next
+      line =~ %r{/bundle/ruby|/usr/local/rvm/} && next
```

Comment:
no one's complained about the lack of sidekiq equivalent, so i didn't bother to chase down whether this should be replaced, and just excised